### ISSUE TITLE
Allow transmission of parent groups in firewall sso

### ISSUE THREAD

**Issue by heiko-ma** [inverse-inc/packetfence#7517]
**Is your feature request related to a problem? Please describe.**
We are using user-based policies on our fortigate and it would be a nice addition if packetfence could pass multiple groups. As it's currently possible to have only one node (and thats perfectly fine), we would like to have a possibility to pass the parent group too. Below you can find our current role structure in packetfence: 
clients <-- This is the default role which all our clients get assigned 
---- marketing-client <-- This is an child role to provide users in this group more rights for internet breakout 
---- developer-client <-- This is an child role to provide users in this group more rights for internet breakout

When users get the developer/marketing-client role assigned, they loose their default role and therefore we would have to change our entire policy set.

**Describe the solution you'd like**
It would be more practical to have a switch in the firewall sso tab to give users the possiblity to transmit the parent roles too. Every current firewall should be able to delimit the groups at a comma or plus sign.

**Comment by nqb** [comment]
Hello @heiko-ma,

I'm not sure to completely understand your need due to verbiage used, you spoke about node, parent group, role but I'm not sure if you are talking about PF or Fortigate.

ATM, PacketFence is sending following information to a Fortigate firewall while doing FW SSO:
- username
- mac
- ip 
- role (equals to current role of node in PacketFence).

You want PacketFence to be able to send:
- current role of node
- parent role(s) of role currently assigned to node
?

Do you have some Fortigate documentation which explain how request should be send to handle several roles ?

Thanks.

**Comment by heiko-ma** [comment]
> You want PacketFence to be able to send:
> 
> - current role of node
> - parent role(s) of role currently assigned to node

Thats correct, I attached the fortigate documentation where the rsso-functionality with multiple groups is exlained:
https://docs.fortinet.com/document/fortigate/7.2.0/new-features/519403/using-a-comma-as-a-group-delimiter-in-radius-accounting-messages

Also below the result on a fortigate when sending two groups:
![image](https://user-images.githubusercontent.com/64192965/217807601-c4538211-6a1e-44df-8c55-856ae9834b9a.png)

You can either delimit with a plus sign or a comma.

For testing I sent a request using radclient:
echo "Acct-Status-Type=Start,Framed-Ip-Address=10.0.0.1,User-Name=aa:bb:cc:dd:ee:ff,Class=access-point+client,Calling-Station-Id=aa:bb:cc:dd:ee:ff" | radclient "address of fortigate" acct "secret-key"
